tools/make/codegen/templates/date_formatter.rs.jinja

```diff
@@ -0,0 +1,143 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
thought: hmmmmm

What I was _hoping_ was that these files could live alongside the rest of their crate (so this would be in capi/src), and you can `cargo make askamagen ffi/capi/src/date_formatter.rs.jinja`. But askama uses compile time templates so that's not ideal.

suggestion: consider still having this file live under ffi/capi, I'm not yet sure which is better but I think you'd have stronger thoughts on this than I. On one hand it's nice to be able to find this when performing find-and-replaces, on the other hand we don't want the file to get lost amongst a sea of rs files. So I genuinely don't know what I prefer, but I wonder if you've had thoughts

Reply:
I wanted to do that initially but askama-derive is fairly opinionated about where the templates live. I can fiddle with it more to see if it will work.

Reply:
ah, that's fine then.

Reply:
symlink?